mindsdb/interfaces/skills/custom/text2sql/mindsdb_sql_toolkit.py

```diff
@@ -52,7 +52,22 @@ def get_tools(self, prefix='') -> List[BaseTool]:
             1. Final Response Format:
                - Assume the frontend fully supports Markdown unless the user specifies otherwise.
                - When the response contains data that fits a table format, present it as a properly formatted Markdown table
-               - Ensure clarity and proper structure for easy readability.
+                   - Ensure clarity and proper structure for easy readability.
```

Comment:
this have to work only for minds, right?

Reply:
If so, should be compose prompt differently, example: https://github.com/mindsdb/mindsdb/pull/10505